modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_benefits_documents_uploader.rb

```diff
@@ -7,6 +7,7 @@ module ClaimsApi
   module V2
     class DisabilityCompensationBenefitsDocumentsUploader < ClaimsApi::ServiceBase
       LOG_TAG = '526_v2_Benefits_Documents_Uploader_job'
+      sidekiq_options expires_in: 48.hours, retry: true
```

Comment:
I'm not fully aware of how `expires_in` behaves but I'd have to imagine that this would not indicate an errored status on the claim upon expiration? I'm not sure if we can set some form of callback or expiration method that would set said status at the end?

TLDR; does this or do we need to find a way to set the claim as `ERRORED`

Reply:
Resolution: Good to enforce 2 day retry across all jobs 👍

Reply:
@FonzMP  I bet we could do that in the [retries exhausted block](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/sidekiq/claims_api/service_base.rb#L19) that gets called when this would expire (but also when anything else exhausts its retries which may be ok?)